### PR TITLE
*ocelot for real, and *meow for tajs.

### PR DESCRIPTION
**Pull request by lewcc**
## What Does This PR Do
Re-implements #20769 now that April Fool's Day is over, for serious consideration.

As per the original body of the PR:
> Adds a new emote, *ocelot, which requires holding two revolvers in your hands. You'll do a long, drawn-out gunshow, before pointing them forwards and firing in a straight line with no spread. If you're one tile away from someone and you're holding two lethal revolvers, you will gib them. Also, tajaran get double damage (ocelots are cats, after all). 

> Also adds a new traitor bundle, Ocelot, which provides two 357 revolvers, a bunch of ammo, holsters, and the appropriate soviet garb. For when you really want something dead, and believe your skill with guns is all you need.

> It can be performed with any two revolvers (even the russian, trick, and arc revolvers), and the different icons will show up in-hand. This means, yes, you can show it off with cap guns.

This version of the PR changes the sound effect to be a slightly edited version of [this](https://freesound.org/people/skymary/sounds/412017/), which is licensed by CC0/Public Domain.

*also...*

Since I had to go and find a good, permissive meow sound, I figure it can't hurt to give tajarans a *meow emote as well. It uses the base version of the sound effect. While I'm *still* at it, I gave *hiss to tajarans as well, since they don't really get much love emote-wise.

## Why It's Good For The Game
It's funny, and if you actually manage to get your hands on two .357 revolvers, you deserve to be able to flex a little.

Regarding the meowing and hissing, tajarans don't get any good custom species emotes, and I think they deserve a bit of love. 

## Images of changes
https://user-images.githubusercontent.com/89928798/229172214-df529984-32dc-4d05-b2bb-fb1df10c5928.mp4
(note that the audio clip at the end has changed).

## Changelog
:cl:
add: New emote, *ocelot, usable with two revolvers in hand.
add: New emote, *meow, a tajaran exclusive.
tweak: Tajaran can now use *hiss as well.
/:cl:

### CODE REVIEW

**Comment by lewcc** [comment]
Very much open to considering the ocelot TC bundle as its own separate PR to keep it more atomic, and to get better ideas for what might go into it.

File: code/datums/status_effects/neutral.dm [hunk 1]
```diff
@@ -143,3 +143,216 @@
...
+	H.visible_message(
+		"<span class='danger'>[owner] begins spinning the revolvers in [owner.p_their()] hands around!</span>",
+	)
+
+/datum/status_effect/revolver_spinning/proc/get_fluff_message(mob/user)
+	var/list/messages = list(
+		"<span class='warning'>[user] tosses one revolver over the other!</span>",
+		"<span class='danger'>[user] flips one revolver around behind [user.p_their()] back and catches it, wow!</span>",
+		"<span class='warning'>[user] spins both revolvers around their fingers skillfully.</span>",
+		"[user] spins one revolver neatly into their pocket, dancing the other around before drawing it out again.",
```
Comment (AffectedArc07) [inline code/datums/status_effects/neutral.dm:247]: No spans?

File: code/modules/mob/living/carbon/human/human_emote.dm [hunk 2]
```diff
@@ -357,6 +357,49 @@
...
 
+/datum/emote/living/carbon/human/ocelot
+	key = "ocelot"
+	key_third_person = "ocelots"
+	hands_use_check = TRUE
+
+/datum/emote/living/carbon/human/ocelot/can_run_emote(mob/living/carbon/human/user, status_check, intentional)
+	. = ..()
+	if(!.)
+		return .
```
Comment (AffectedArc07) [inline code/modules/mob/living/carbon/human/human_emote.dm:368]:
```suggestion
		return
```

**Comment by lewcc** [comment]
I'm gonna repackage this

**Pull request closed without merging**